NuGet.Docs/Contribute/Setting-up-the-NuGet-Development-Environment.md

```diff
@@ -12,7 +12,7 @@ After VS2015 has been installed, you would need to:
 To build NuGet.VisualStudioExtension, execute the following steps:
 
 1. Add the directory of msbuild 14, e.g. C:\Program Files (x86)\MSBuild\14.0\Bin, to PATH
-1. Download the latest version of [NuGet.exe](www.nuget.org/nuget.exe), and add its directory to PATH
+1. Download the latest version of [NuGet.exe](http://www.nuget.org/nuget.exe), and add its directory to PATH
```

Comment:
The "nuget.exe" download is available in "http://www.nuget.org/nuget.exe" or "https://www.nuget.org/nuget.exe". What is important is the usage of the absolute URI because the current url is translate to ""http://docs.nuget.org/contribute/www.nuget.org/nuget.exe" that end in a 404.